### PR TITLE
10246_update_expiration_date_time_logic

### PR DESCRIPTION
**Pull request by ozamani9**
*Issue #:*/bcgov/entity#10246

*Description of changes:*

Change the expirdation_date_time to 23:59pm instead of 00:01am

### CODE REVIEW

File: client/src/store/index.js [hunk 1]
```diff
@@ -1945,22 +1945,22 @@ export const mutations = {
     state.nrData.lastUpdate = state.lastUpdate
 
     if (state.consentDateForEdit) {
-      let date = new moment(state.consentDateForEdit + ' 00:01:00')
+      let date = new moment(state.consentDateForEdit + ' 23:59:00')
```
Comment (severinbeauvais) [inline client/src/store/index.js:1948]: In the ticket (10246) it mentions that consent date might need to be updated. Did you confirm with Linda? And I am not sure where else this date is used.

File: client/test/unit/specs/transactions.spec.js [hunk 2]
```diff
@@ -312,11 +312,11 @@ describe('Testing Transactions.vue EXPIRED', () => {
     // set NR info
     vm.$store.commit('setNrInfo', {
       additionalInfo: 'The quick brown fox...',
-      consent_dt: '2021-08-01T07:01:00+00:00',
+      consent_dt: '2021-08-01T06:59:00+00:00',
       consentFlag: 'R',
       corpNum: 'BC1234567',
       entity_type_cd: null,
-      expirationDate: '2021-09-01T07:01:00+00:00',
+      expirationDate: '2021-09-01T06:59:00+00:00',
```
Comment (severinbeauvais) [inline client/test/unit/specs/transactions.spec.js:319]: Are there any tests in here for PST offset (8h) ?

And maybe there should be tests for the edge cases (ie, [the dates the time zone changes](https://en.wikipedia.org/wiki/Pacific_Time_Zone#:~:text=Effective%20in%20the%20U.S.%20in,the%20first%20Sunday%20in%20November.)).

Comment (ozamani9) [reply client/test/unit/specs/transactions.spec.js:319]: no there werent any tests before for PST offset, or time zone changes

Comment (ozamani9) [reply client/test/unit/specs/transactions.spec.js:319]: time zone changes will be difficult to test, because they appear on different days every year, so i would leave that one... but i can update one of the tests to include 07:59 instead of 06:59 if you would like

Comment (severinbeauvais) [reply client/test/unit/specs/transactions.spec.js:319]: You don't need a generic test for all possible change dates -- you can add tests for date changes in late 2021 and early 2022 :)

File: client/src/store/index.js (hunk 1 above)
Comment (ozamani9) [reply client/src/store/index.js:1948]: consent date needs to be updated as well.  Linda chatted with Debbie, and they want the expiry dates to be on the previous day at 23:59.    2 minutes before the 00:01:00.   so i need to fix this in the update.

Comment (severinbeauvais) [reply client/src/store/index.js:1948]: Wait, what? I just chatted with Linda, who said the consent time doesn't matter, just the date.

As for expiry date... the previous day? What do you mean -- what needs to change?

**Comment by ozamani9** [comment]
a new branch and P.R. will be created for this work
